Tests/UnitTests/Paywalls/Components/ButtonComponentTests.swift

```diff
@@ -11,7 +11,12 @@ class ButtonComponentCodableTests: TestCase {
         "type": "stack",
```

Comment:
Are the changes in this file (below) related? If so, how? 😄

Reply:
@JayShortway Nope! Just tests that didn't run on CI because of the feature/compiler flag so didn't see they failed a while back 🫠